vboxlauncher@mockturtl/files/vboxlauncher@mockturtl/applet.js

```diff
@@ -52,7 +52,7 @@ MyApplet.prototype = {
 
     try {
       this.set_applet_icon_name(ICON)
-      
+
```

Comment:
Ideally whitespace fixes should be in a separate commit, but it's OK.  

If you want to level up your git, you can practice:

```sh
$ git reset HEAD~  # bring the working directory to the state before this commit
$ git add --patch applet.js  # select changes that go together, instead of the whole file
                             # then commit as usual
```

Reply:
You completely right, sorry, I rollback this kind of changes !